lib/graphql/subscriptions/event.rb

```diff
@@ -53,6 +53,34 @@ def fingerprint
 
       class << self
         private
+
+        # This method does not support cyclic references in the Hash,
+        # nor does it support Hashes whose keys are not sortable
+        # with respect to their peers ( cases where a <=> b might throw an error )
+        def deep_sort_hash_keys(hash_to_sort)
+          raise ArgumentError("Argument must be a Hash") unless hash_to_sort.is_a?(Hash)
+          hash_to_sort.keys.sort.map do |k|
+            if hash_to_sort[k].is_a?(Hash)
+              [k, deep_sort_hash_keys(hash_to_sort[k])]
+            else
+              [k, hash_to_sort[k]]
+            end
```

Comment:
It seems like, if `hash_to_sort[k]` is an array containing hashes, then we _want_ to keep going here, right? 

```suggestion
            elsif hash_to_sort[k].is_a?(Array)
              [k, deep_sort_array_hashes(hash_to_sort[k])
            else
              [k, hash_to_sort[k]]
            end
```